saleor/product/models.py

```diff
@@ -469,6 +472,12 @@ def get_absolute_url(self):
             'product:collection',
             kwargs={'pk': self.id, 'slug': self.slug})
 
+    @property
+    def is_available(self):
+        today = datetime.date.today()
```

Comment:
Couldn't we have this instruction in a single condition line? That would save some CPU cycles when `is_published is True`.